assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -100,6 +100,11 @@ angular.module("openshiftConsole")
             scope.buildConfig.contextDir = annotations.sampleContextDir || "";
           };
 
+          scope.usingSampleRepo = function() {
+            var sampleRepo = _.get('scope.image.metadata.annotations.sampleRepo');
```

Comment:
told you wrong before, its` _.get(scope, 'image.metadata.annotations.sampleRepo')`